tests/acceptance/features/bootstrap/FeatureContext.php

```diff
@@ -107,6 +107,43 @@ public function __construct(
 		$this->cookieJar = new CookieJar();
 	}
 
+	/**
+	 * When we run API tests in CI, the user file system frequently does not get deleted.
+	 * So user is created with retry in order to comeback the flakiness in CI
+	 *
+	 * @param string $user
+	 * @param array<mixed> $userAttributes
+	 *
+	 */
+	private function createUserWithRetry(string $user, array $userAttributes): void {
+		$retryCreate = 1;
+		$isUserCreated = false;
+		while ($retryCreate <= 5) {
```

Comment:
```suggestion
		while ($retryCreate <= 5) {
```

Can we iterate it only for one time, when there is only 400 status code

Reply:
no lets check for and retry for others as well  other than status code 200. Any how we want to create a user.